client/app/telecom/telephony/alias/special/rsva/telecom-telephony-alias-special-rsva.html

```diff
@@ -0,0 +1,314 @@
+<section class="telephony-group-alias-special-rsva">
+
+    <header>
+        <section-back-link section-back-link-to-state="telecom.telephony.alias.special"
+                           section-back-link-title="{{ 'telephony_alias_special_rsva_title_back_link' | translate }}">
+        </section-back-link>
+        <h1 data-translate="telephony_alias_special_rsva_title"></h1>
+    </header>
+
+
+    <div class="telecom-section-content"
+         data-ng-if="!AliasSpecialRsvaCtrl.isLoading">
+
+        <toast-message></toast-message>
+
+        <div class="row">
+            <div class="col-md-8 widget-presentation">
+                <div class="alert alert-info"
+                     role="alert">
+                    <span data-translate="telephony_alias_special_rsva_topology_info"></span>
```

Comment:
Apply `data-translate` directly on your parent element or you can use `<oui-message>` :)